scripts/test-macos-enterprise-packaging.sh

```
@@ -192,77 +162,111 @@ wheel_gid="$(stat -f '%g' /Library)"
-grep -Fq "existing DefenseClaw installation detected" "${fixture}/acl.stderr" || fail "ACL-state refusal was not explicit"
-grep -Fq "no changes were made" "${fixture}/acl.stderr" || fail "ACL-state refusal did not attest no changes"
-[ "$(sudo -n shasum -a 256 "$config_dest" | awk '{print $1}')" = "$config_hash_before_refusal" ] || fail "ACL-state refusal modified managed config"
-[ "$(sudo -n shasum -a 256 "$gateway_dest" | awk '{print $1}')" = "$gateway_hash_before_refusal" ] || fail "ACL-state refusal modified gateway binary"
-sudo -n chmod -N "$managed_root"
-
-# Even damaged installed metadata must not turn this package installer into an
-# implicit repair/upgrade path. It refuses before replacing state; recovery is
-# owned by the staged managed upgrader.
-sudo -n chown "$(id -u):$(id -g)" "$config_dest"
-sudo -n chmod 0666 "$config_dest"
+    --no-start >"${fixture}/reinstall.stdout" 2>"${fixture}/reinstall.stderr" \
```

Shellcheck (0.11.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)